### DISCUSSION TITLE
Create a FrankenPHP variation

### DISCUSSION THREAD

**Discussion by jaydrogers** [serversideup/docker-php#220] [Ideas & Feature Requests]
# 👉 Describe the problem
[comment]: <> (What are you trying to solve?)

Traditional deployment methods of PHP can be tricky because it usually requires two processes to run a PHP app server.

# 🏆 How to solve this problem
[comment]: <> (Describe the feature that you are proposing and how it will solve the problem)

### Use FrankenPHP app server

FrankenPHP is a modern PHP app server that simplifies the deployment of PHP. More detail about the project is here: https://frankenphp.dev/

Some additional discussion can be located here https://twitter.com/caddyserver/status/1716564187781136855

### Adapt the “serversideup/php” images UX to this variation
We can use the same environment variables and allow users to configure their PHP images easily like we do with the rest of our images.

# 💯 How do we validate the problem is solved?
[comment]: <> (Explain use cases on how we can measure the success of this implementation)

Users will be able to deploy a Laravel PHP application running FrankenPHP with the same customization experience as the other serverisdeup/php images.

**Comment by ijpatricio** [comment]
This is bombastic 💯

**Comment by ijpatricio** [comment]
Especially (as it seems to me) if using Laravel Octane is optional!

**Comment by herpaderpaldent** [comment]
i'm most curious how frankenphp will benchmark compared to your other images

**Comment by jaydrogers** [comment]
FrankenPHP has been moved into development 🥳

Track its status here: https://github.com/serversideup/docker-php/pull/283
